### PR TITLE
fix: use dist to release package

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Review by dimakis — APPROVED** [review]

File: spectral/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "build": "npm run clean && tsc && npm run copy-ruleset",
     "clean": "rm -rf dist",
     "lint": "eslint src/**/*.ts",
-    "copy-ruleset": "cp ./ruleset.yaml ./dist/ruleset.yaml && cp ./package.json ./dist/package.json",
+    "copy-ruleset": "cp rhoasapi ruleset.yaml package.json ./dist ",
```
Comment (dimakis) [inline spectral/package.json:11]: copying the binary here?

Comment (wtrocki) [reply spectral/package.json:11]: yes. it needs to be in the dist

**Comment by wtrocki** [comment]
To verify it I have used `cd dist && npm link .`

File: spectral/package.json (hunk 1 above)
Comment (dimakis) [reply spectral/package.json:11]: ok, I did that before and you didn't like it?

**Pull request merged**